### DISCUSSION TITLE
Cannot read property 'TEXT_NODE' of undefined

### DISCUSSION THREAD

**Discussion by Tonysmark** [niklasvh/html2canvas#2695] [Q&A]
![图片](https://user-images.githubusercontent.com/22767695/132486664-fd4d4bc5-0bd5-455d-82fb-389c7a5200ea.png)
![图片](https://user-images.githubusercontent.com/22767695/132486844-37d4245d-a8be-436e-973e-97aa71c35376.png)
![图片](https://user-images.githubusercontent.com/22767695/132488979-f4376adc-0e10-4c09-8803-fd9cca31009d.png)

    "@tarojs/components": "3.3.5",
    "@tarojs/react": "3.3.5",
    "@tarojs/runtime": "3.3.5",
    "@tarojs/taro": "3.3.5",
    "html2canvas": "^1.3.2",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",

**Comment by msnmask** [comment]
you solved it yet human ?

**Reply by Tonysmark** [reply]
not at all
